<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoiceDialogWnd</class>
 <widget class="QWidget" name="VoiceDialogWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(16, 16, 16);
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn_2">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	background:url(:/res/res/CloseWndM1.png);
	background-color:transparent;
	border: none;
}
QPushButton:hover
{
	background:url(:/res/res/CloseWndM2.png);
	background-color:transparent;
}
QPushButton:pressed
{
	background:url(:/res/res/CloseWndM3.png);
	background-color:transparent;
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(250, 250, 250);
border-radius: 15px;</string>
     </property>
     <property name="text">
      <string>XXX请求通话</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>24</number>
     </property>
     <property name="leftMargin">
      <number>12</number>
     </property>
     <property name="rightMargin">
      <number>12</number>
     </property>
     <property name="bottomMargin">
      <number>12</number>
     </property>
     <item>
      <widget class="QPushButton" name="agreeBtn">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/res/res/AgreeBtn.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disAgreeBtn">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/res/res/DisAgreeBtn.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/res/res/HangUpBtn.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
